core/src/main/kotlin/org/evomaster/core/Main.kt

```diff
@@ -201,10 +186,41 @@ class Main {
                     info("Passed time (seconds): ${stc.getElapsedSeconds()}")
                     info("Execution time per test (ms): ${stc.averageTestTimeMs}")
                     info("Computation overhead between tests (ms): ${stc.averageOverheadMsBetweenTests}")
-                    val timeouts = data.find { p -> p.header == Statistics.TEST_TIMEOUTS }!!.element.toInt()
-                    if (timeouts > 0) {
-                        info("TCP timeouts: $timeouts")
+                }
+            }
+
+
+            if(config.security){
+                //apply security testing phase
+                LoggingUtil.getInfoLogger().info("Starting to apply security testing")
+
+                //TODO might need to reset stc, and print some updated info again
+
+                when(config.problemType){
+                    EMConfig.ProblemType.REST -> {
+                        val securityRest = injector.getInstance(SecurityRest::class.java)
+                        solution = securityRest.applySecurityPhase()
```

Comment:
This line:

solution = securityRest.applySecurityPhase()

overrides the variable solution, which contains the set of tests other than security tests.